grace/models/README.md

```diff
@@ -0,0 +1,114 @@
+# Graph Attributes for Classification
+
+## Node features:
+
+The nodes in the constructed graph are processed via an image feature extractor model, _e.g._ `ResNet-152` pre-trained with `ImageNet` weights, to crop a small patch centred at the node's (`x`, `y`) coordinates as the specified shape (_i.e._ `bbox_size`).
+
+The node-corresponding patches are then run through the forward pass of the model, which has its last fully-connected (FC) classification layer chopped of to return a 1D vector of image embeddings. The embedding dimension depends on the feature extractor of choice; for instance, the `ResNet-152` will output 2,048 features-long vector. These embeddings provide an **hypothesis-free approach** to describe the images without incorporation of user's prior about what is or isn't important to featurize in the image patch data. In GRACE, the node features are stored as `GraphAttrs.NODE_FEATURES` array.  
+
+
+## Edge properties:
```

Comment:
It might be useful to be clear about which properties are actually used in the training, we know the `raw` properties can't be passed directly to the network because it would memorise it, so lets make sure the user knows that.

Reply:
Making an issue from this comment + adding this into the documentation before merging 🚀

Reply:
Done, detailed explanation included (although this may change in an upcoming new PR)... Thank you for the suggestion @crangelsmith 👍